extensions/libraries/redcore/form/fields/rlimitbox.php

```diff
@@ -86,7 +86,13 @@ protected function getOptions()
 			// Add an option to show all?
 			$showAll = isset($this->element['showall']) ? ($this->element['showall'] == "true") : true;
 
-			if ($showAll)
+			// The page dies on too big request, make sure it doesn't happen
+			$itemModel   = \Aesir\Model\AdminModel::getAdminInstance('Items', array('ignore_request' => true), 'com_reditem');
```

Comment:
you should not use Aesir classes in redCORE, since some sites might have redCORE but without Aesir extensions